modules/va_forms/app/workers/va_forms/form_builder.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'va_forms/regex_helper'
+
+module VAForms
+  class FormBuilder
+    include Sidekiq::Worker
+    include SentryLogging
+    FORM_BASE_URL = 'https://www.va.gov'
+
+    def perform(form)
```

Comment:
Since the `form` argument is a hash with multiple keys, let's document the shape of it similar to the [AppealReceivedJob](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/app/workers/appeals_api/appeal_received_job.rb#L11-L21).